addons/craft/create_recipes.py

```diff
@@ -1,4 +1,7 @@
-import urllib2
+try:
+    import urllib.request as urllib2
+except ImportError:
+    import urllib2
```

Comment:
As far as I know `urllib2` no longer exists in Python 3, so why would importing it work in the exception case?

Reply:
It's supposed to be backwards compatible with python2 although I haven't tested this.  I can give it a test and see if it actually works.

Reply:
I was able to test this and with a few additional changes that are now part of a PR, it works on python 2 and 3.  One issue I ran into on python2 was getting hit with capcha whereas I didn't with python3.  I'm not sure if that has anything to do with the changes though as that part of the code is untouched.  I was able to test python2 by running the code on python3 first so that the html files were downloaded without issue, and then running it again in the python2 environment.

Reply:
I'm not a fan of messing the code up to support both versions, Python 2 deserves to die already :P But I won't force you to change it back, I'll leave it up to you.